src/components/App.tsx

```diff
@@ -451,6 +463,11 @@ class App extends Component<IProps, IState> {
                   <b>Balance: </b><span>{balance}</span>
                 </div>
               )}
+              {
+                <div className="app-register">
+                  <Button disabled={this.state.appRegistered} onClick={this.handleAppRegister} >Register App</Button>
```

Comment:
Do not keep disabled button when the app is authenticated. Show authenticated status in account section.